avm/ptn/dev-ops/cicd-agents-and-runners/main.bicep

```diff
@@ -282,7 +282,7 @@ module acr 'br/public:avm/res/container-registry/registry:0.9.3' = {
           {
             subnetResourceId: networkingConfiguration.networkType == 'createNew'
               ? filter(
-                  newVnet!.outputs.subnetResourceIds,
+                  newVnet.?outputs.subnetResourceIds ?? [],
```

Comment:
Hey @johnlokerse,
I'd be curious why change this to a tryGet() if the case is only execute if the `networkType == 'createNew'`? Wouldn't the `!` (i.e., it won't be `null`) be more accurate (yet alone requiring the fallback)? Or is Bicep executing the function regardless, hence leading to errors.

Reply:
Hey @AlexanderSehr, suppressing would be fine here due to the conditions, and we know `newVnet` has a value anyway so there is no need to use any safe-dereferencing here. Same thing for NAT GW. Btw, I didn't do any validation, my answer is based on theory.. 😄

Reply:
Also, this is not my change so I don't know the reasoning behind it

Reply:
How odd. It showed it up as your change before, but now not anymore. Let me resolve the comment then (even though my point still stands @sebassem). Both works eventually though, so it's nothint to prioritize.